docs/DeveloperGuide.md

```diff
@@ -397,15 +403,15 @@ otherwise)
 * 1a. User inputs a negative number.
 
     * 1a1. StoreMando shows an error message.
-    
+
     * 1a2. StoreMando prompt the user for a correct input.
```

Comment:
Should this be "StoreMando prompts the user for the correct input"?

Reply:
Yup thanks Fazil!